### ISSUE TITLE
JAWS causing Material UI autocomplete combobox odd behaviour

### ISSUE THREAD

**Issue by haze0028** [FreedomScientific/VFO-standards-support#630]
### Summary

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Open JAWS
> 1. Go to [https://uxdc.pages.cloud.statcan.ca/daaas/daaas-prototypes/#/vetting-app/request-researcher](https://uxdc.pages.cloud.statcan.ca/daaas/daaas-prototypes/#/vetting-app/request-researcher)
> 2. Tab to "Request folder path" combobox
> 3. Use arrow keys to scroll through options (notice focus goes to next input label)
> 4. Select an option with mouse
> 5. Tab from combobox to the "Clear (X)" button (notice JAWS reads page details in instead of "Clear button")

### Expected result

> 1. Use arrow keys should allow user to scroll through combobox options.
> 2. Tabbing to "Clear" button should read the button, instead of page details.

### Actual result

> 1. Using the arrow keys when combobox is selected shifts focus to next input label.
> 2. Tabbing to Clear button reads page details.

### Examples
[Without JAWS](https://watch.screencastify.com/v/cndk2hoeLIncJdHFvyuk)

[With JAWS](https://watch.screencastify.com/v/zi9czQBHOX5v1t5ystUl)

**Comment by stevefaulkner** [comment]
@haze0028 I am getting a timeout on https://uxdc.pages.cloud.statcan.ca/daaas/daaas-prototypes/#/vetting-app/request-researcher

**Comment by haze0028** [comment]
> @haze0028 I am getting a timeout on https://uxdc.pages.cloud.statcan.ca/daaas/daaas-prototypes/#/vetting-app/request-researcher

I've updated the link. It should work now.

**Comment by JAWS-test** [comment]
I don't think this is a JAWS error, but the combo box is implemented incorrectly. So the label and the clear button are inside the combo box. This leads to no output at all when reading with the virtual cursor. I recommend a implementation according to https://w3c.github.io/aria-practices/#combobox
